packages/medusa/src/api/routes/store/payment-collections/manage-payment-session.ts

```diff
@@ -0,0 +1,51 @@
+import { IsInt, IsNotEmpty, IsOptional, IsString } from "class-validator"
```

Comment:
question: Should we rename this endpoint to `set-payment-sessions.ts`?

Reply:
or rename to `manage-payment-sessions.ts` in plural?

Reply:
will rename it to `manage-payment-sessions.ts` as it handles adds, updates and deletes